homeassistant/components/lovelace/const.py

```diff
@@ -75,6 +76,8 @@
     # For now we write "storage" as all modes.
     # In future we can adjust this to be other modes.
     vol.Optional(CONF_MODE, default=MODE_STORAGE): MODE_STORAGE,
+    # Set to allow adding dashboard without hyphen
```

Comment:
Should we make a note here that we expect this to be only set by system generated configurations and that we do not feel the need to consider custom panels if we want to add a new panel in the future?